app/javascript/stylesheets/globals.scss

```diff
@@ -15,20 +18,188 @@ $background-color: #F9F6F2;
 
 $background-color-dark: darken($background-color, 5%);
 
+
 @mixin default-button{
-  border-color: lighten($gray-color-light, 5%);
-  transition: none;
-  &, &:visited {
-    color: $gray-color-dark;
+  // border-color: lighten($gray-color-light, 5%);
+  // transition: none;
+  // &, &:visited {
+  //   color: $gray-color-dark;
+  // }
+  // &:hover {
+  //   background-color: $secondary-color-light;
+  //   border-color: lighten($secondary-color, 10%);
+  //   color: $dark-color;
+  // }
+  // &:active, &.active {
+  //   background-color: $secondary-color;
+  //   border-color: darken($secondary-color, 10%);
+  //   color: white;
+  // }
+}
```

Comment:
May want to leave this default button mixin here in case it's used elsewhere on the site.